### ISSUE TITLE
Colorlib has taken over AdminLTE?

### ISSUE THREAD

**Issue by TorbenLundsgaard** [ColorlibHQ/AdminLTE#2081]
Obviously Colorlib has taken over AdminLTE. What's going here? @almasaeed2010

**Comment by GregJPreece** [comment]
On what is your question based?

**Comment by puikinsh** [comment]
Yes, Colorlib has taken over this project. It will be maintained and supported like before.

**Comment by GregJPreece** [comment]
Thanks for the info, @puikinsh, I must have missed that. If you wouldn't mind answering a couple of questions for the AdminLTE faithful:

- Will v3 see a final release version?
- Is there any intention to turn AdminLTE into a premium theme, or will it remain free?

**Comment by puikinsh** [comment]
It  is close to a fonal relase. If anyone could jump i to testing it would make thing much faster
https://github.com/ColorlibHQ/AdminLTE/tree/v3-dev

It will remain free just like some 600 of other our templates and themes. In case you haven’t heard about our templates: 
https://colorlib.com/wp/templates/

**Comment by InToSSH** [comment]
Glad to hear v3 is close to release, I am currently testing the v3-dev, as I wanted to build a project on Bootstrap 4, so far so good. When migrating from v2 I had to rewrite the sidebar and topnav, and replace box with card.
Only issue (maybe) I have noticed is in the `nav-sidebar` when using `nav-header`.
When the `navbar-header` is the first child element of `<ul class="nav nav-sidebar flex-column">` it has 0.5rem padding:
```
.nav-sidebar .nav-header {
    font-size: .9rem;
    padding: 0.5rem;
```  

But the rest of them have:
```
.nav-sidebar .nav-header:not(:first-of-type) {
    padding: 1.7rem 1rem .5rem 1rem;
```

So if I use the `nav-header` to section my menu and I use it right after the `ul` it is not aligned with the rest of the `nav-header`s. Not sure if this is intended or not, for now I fixed it with BS4 class `pl-3`.
